build.rs

```diff
@@ -171,6 +212,29 @@ fn build_v8() {
   build("rusty_v8", None);
 }
 
+fn checkout(dest: &str, tag: &str) {
+  assert!(Command::new("git")
+    .arg("checkout")
+    .arg(tag)
+    .current_dir(dest)
+    .status()
+    .unwrap()
+    .success());
+}
+
+fn maybe_clone_repo(dest: &str, repo: &str) {
+  if !Path::new(&dest).exists() {
+    assert!(Command::new("git")
+      .arg("clone")
+      .arg("--depth=3")
```

Comment:
Out of curiosity, why --depth=3?

Reply:
It's a workaround to later checkout to the ndk 22 commit https://github.com/denoland/rusty_v8/blob/894374a1bfe4de6fc1dde653c766b4fefec50de4/build.rs#L182-L187

Reply:
Andd i now realize how dumb this is. It's going to break when there is a new commit :(

Reply:
fixed